src/main/java/com/google/collegeplanner/data/Course.java

```diff
@@ -0,0 +1,127 @@
+package com.google.collegeplanner.data;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+public class Course {
+  private String courseId;
+  private String name;
+  private String semester;
+  private int credits;
+  private String deptId;
+  private String description;
+  private String[] gradingMethod;
+  private String coreqs;
+  private String prereqs;
+  private String restrictions;
+  private String additionalInfo;
+  private String creditGrantedFor;
```

Comment:
Consider having a short description of these variables, e.g. What would be the format or any example if needed. You have mentioned the format of semester below, I think it is better to make variables documented while you define them.